Week1/Day1/ExercisesXPGold.py

```diff
@@ -0,0 +1,16 @@
+#Exercise 1
+print("Hello world\n" * 4 + "I love python\n" * 4)
+
+#Exercise 2
+season = int(input("Please input a month from 1 to 12: "))
+
+if season == 12 or season == 1 or season == 2:
```

Comment:
must be a more elegant way to check if a number is inside range...

Reply:
season = int(input("Please input a month from 1 to 12: "))
winter = [12, 1, 2]
spring = range(3, 6)
summer = range(6, 9)
autumn = range(9, 12)

if season in winter:
    print("This is a Winter season.")
elif season in spring:
    print("This is a Spring season.")
elif season in summer:
    print("This is a Summer season.")
elif season in autumn:
    print("This is a Autumn season.")
else:
    print("The number is out of the range. Please try again.")